sormas-api/src/main/resources/captions.properties

```diff
@@ -2962,6 +2962,7 @@ specailCaseAccessNoAccessGranted=No special access granted for this case
 SelfReport = Self report
 SelfReport.type = Type
 SelfReport.reportDate = Report date
+SelfReport.caseReference = Case Reference
```

Comment:
please change to "Case reference number"

Reply:
changed